main.py

```diff
@@ -2,6 +2,52 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for genre in genres:
+        Genre.objects.create(name=genre, )
+
+    for first_name, last_name in actors:
+        Actor.objects.create(
+            first_name=first_name,
+            last_name=last_name,
+        )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(
+        name="Drama"
+    )
+    Actor.objects.filter(
+        first_name="George"
+    ).update(
```

Comment:
It works here, but it's better to search by the first name and the last name according to the task. The same in other similar cases

![image](https://user-images.githubusercontent.com/104218550/223810409-ca74032a-231b-4e6d-ad40-8aa485af9d4a.png)